app/public/utils/utils.js

```diff
@@ -22,7 +22,7 @@ export const reduceSerialIf = (
 
 export function getReadableFileSizeString(fileSizeInBytes) {
     let i = -1;
-    const byteUnits = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB'];
+    const byteUnits = [' kB', ' MB', ' GB', ' TB', ' PB', ' EB', ' ZB', ' YB'];
     do {
```

Comment:
I think that addition of space before units should be than in other place and be automated

Reply:
Done